example/static/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>landline-api test page</title>
+    <!-- I'm assuming here that Promises and XHR2 are avilable. Most notable, IE doesn't yet support promises, although they're currently in the technical preview. -->
```

Comment:
I don't think the browser you use to view this testing page really matters, as it is geared more towards testing the API running on the server, not about testing the javascript. landline-web is what should be tested cross-browser.